Libraries/LibWeb/Internals/InternalGamepad.h

```diff
@@ -7,8 +7,55 @@
 #pragma once
 
 #include <LibWeb/Bindings/PlatformObject.h>
-#include <SDL3/SDL_gamepad.h>
-#include <SDL3/SDL_joystick.h>
+
+// NOTE: Unfortunately we have to just manually define these enums given forward declaring is not supported, so updating SDL versions that adjust the content of these enums may break things
+
+typedef enum SDL_GamepadButton {
```

Comment:
suggestion: It may be better to make buttons, axes and triggers return `Array<i32, [length]>` instead. It doesn't have to be super pretty with being an internal testing class not exposed to the web.